sky/provision/aws/instance.py

```diff
@@ -173,17 +175,17 @@ def _create_instances(ec2_fail_fast, cluster_name: str, node_config: Dict[str,
     per_subnet_tries = max_tries // num_subnets
     for i in range(max_tries):
         try:
-            if 'NetworkInterfaces' in conf:
```

Comment:
Probably we can add `assert not conf.get('NetworkInterfaces'), conf`?

Reply:
Done.